### DISCUSSION TITLE
wslu 4.1.2

### DISCUSSION THREAD

**Discussion by car-meows** [wslutilities/wslu#304] [General]
- feat: [#293] allow set path with param -e to set BROWSER variable
- fix: [#264] make DESTDIR properly configured in header 
- fix: [#268] allow skipping validation (with #301 by @JaZo)
- fix: [#290] use proper folder for the states
- fix: [#291] allow disable reading from registry
- fix: [#294] add bash version requirement.
- fix: [#295] decode file protocol url properly
- fix: sysdrive_prefix returning wrong label when Windows installed at label other than C: (@epiciskandar)
- fix: replace sed with simple basename (@epiciskandar)
- chore: Add support for Rocky to installer (@cjonesy)
- fix(wslact): fix memory-reclaim incoherence (@kbpk)
- fix(wslact): increase memory reclamation amount (@kbpk)
- fix(wslview): fix opening links with brackets (@miduddin)
- docs: fixed typos (@Smoothieewastaken)
- ci: Deploy to Pengwin debian bookworm as well (@crramirez)
- ci: upgrade CI pengwin deployment ruby version

<hr /><em>This discussion was created from the release <a href='https://github.com/wslutilities/wslu/releases/tag/v4.1.2'>wslu 4.1.2</a>.</em>

**Comment by JaZo** [comment]
Thanks for this release! I'm on Ubuntu, but the release isn't available yet on the [PPA version](https://launchpad.net/~wslutilities/+archive/ubuntu/wslu). Do I need to wait a little longer?

**Reply by car-meows** [reply]
Yes, unfortunately I lost my old Ubuntu instance to submit Ubuntu packages, I am currently trying to set it up again, it will definitely take a longer time to build a PPA version

**Reply by JaZo** [reply]
Thanks for the reply, I'll wait and see. Good luck with the build!
